deployment.rb

```diff
@@ -82,7 +82,7 @@ def load_configuration()
     'assets_s3_directory'         => rack_env == :production ? 'assets' : "assets_#{rack_env}",
     'sources_s3_bucket'           => 'cdo-v3-sources',
     'sources_s3_directory'        => rack_env == :production ? 'sources' : "sources_#{rack_env}",
-    'netsim_api_publickey'        => [:development].include?(rack_env) ? 'JGW2rHUp_UCMW_fQmRf6iQ==' : 'HQJ8GCCMGP7Yh8MrtDusIA==',
+    'netsim_shard_expire_time'    => 7200,
```

Comment:
How about calling this netsim_shard_expiry_seconds  (to avoid someone getting the units wrong)